### PR TITLE
Super Duper Vanilla support

### PR DESCRIPTION
**Pull request by IMS212**
cc @Eldeston

yeah this is a thing

### CODE REVIEW

File: src/main/java/net/coderbot/iris/uniforms/HardcodedCustomUniforms.java [hunk 1]
```diff
@@ -132,4 +143,31 @@ private static float getBlindFactor() {
 	private static float frac(float value) {
 		return java.lang.Math.abs(value % 1);
 	}
+
+	private static float getAdjTime() {
+		return (float) Math.abs((((((float) WorldTimeUniforms.getWorldDayTime()) / 1000.0) + 6.0) % 24.0) - 12.0);
+	}
+
+	private static float getDay() {
+		return (float) Math.clamp(5.5 - getAdjTime(), 0.0, 1.0);
```
Comment (Eldeston) [inline src/main/java/net/coderbot/iris/uniforms/HardcodedCustomUniforms.java]: Could we change this back to 5.4?

Comment (IMS212) [reply src/main/java/net/coderbot/iris/uniforms/HardcodedCustomUniforms.java]: Sure.

**Review by coderbot16 — APPROVED** [review]
Fair enough, I don't want to add a ton of hardcoding but this seems simple enough for now

**Comment by coderbot16** [comment]
Conflicts with trunk though.

**Comment by IMS212** [comment]
Fixed.

**Review by coderbot16 — APPROVED** [review]
Thanks!

**Pull request merged**